blocks/init/src/Blocks/wrapper/components/wrapper-editor.js

```diff
@@ -20,6 +20,9 @@ export const WrapperEditor = ({ attributes, setAttributes, children }) => {
 	const wrapperOffsetLarge = checkAttr('wrapperOffsetLarge', attributes, manifest, true);
 	const wrapperWidthLarge = checkAttr('wrapperWidthLarge', attributes, manifest, true);
 
+	// First letter for WrapperTag is capitalized on purpose. That way it can be used as a dynamic tag.
+	const WrapperTag = checkAttr('wrapperTag', attributes, manifest);
```

Comment:
Can you please check with the JS team if this is legit to do?

Reply:
They say that a better approach would be using `createElement`, but the one I wrote should also be okay. We can wait for a while to see if anyone else replies and then decide.

Reply:
I'm fine with it staying as it is, so if everyone else agrees, this is good to go 😄